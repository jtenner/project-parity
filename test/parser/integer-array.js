const { OptionType } = require("../../lib");

module.exports = {
  argv: ["--option-a", "10,9,8,7,6"],
  config: {
    optionA: {
      type: OptionType.IntegerArray,
      default: [],
    },
  },
};
