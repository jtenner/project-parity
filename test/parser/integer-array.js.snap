{
  args: [],
  cliProvided: Set(1) { 'optionA' },
  configProvided: Set(0) {},
  errors: [],
  options: { optionA: [ 10, 9, 8, 7, 6 ] },
  unknown: []
}
